docs/_articles/en/soql/soql-builder.md

```diff
@@ -44,10 +46,15 @@ Launch SOQL Builder from within a Salesforce DX project. How you launch SOQL Bui
 
 DX projects have a sample `accounts.soql` file in the `<project-folder>/scripts/soql` directory. However, you can create and store your `.soql` files in any directory.
 
+<!-- ekapner, 2/18, doc update -->
+
 1. (If necessary) Create a `.soql` file.
-1. Right-click the file name, select **Open With**, then **SOQL Builder**.
+1. Click on the `.soql` file.
+1. Click the Switch Between SOQL Builder and Text Editor.
```

Comment:
The `the Switch Between SOQL Builder and Text Editor.` seems weird to me without referencing _what_ it is, such as an icon or button. If They can't see the gif then the reader will have no idea what to look for.

Reply:
Called it an icon.